src/ape/api/address.py

```diff
@@ -72,6 +84,13 @@ def __dir__(self) -> List[str]:
         ]
 
     def __repr__(self) -> str:
+        """
```

Comment:
I don't think we need to document `repr`, or at least I haven't been

Reply:
No prob, I'll pull it out.